lib/client/index.ts

```diff
@@ -280,8 +286,10 @@ export class LibraClient {
           new LibraTransactionResponse(
             new LibraSignedTransaction(transaction, sender.keyPair.getPublicKey(), senderSignature),
             response.getValidatorId_asU8(),
-            response.getAcStatus(),
-            response.getMempoolStatus(),
+            response.getAcStatus() === undefined ? -1 : (response.getAcStatus() as AdmissionControlStatus).getCode(),
+            response.getMempoolStatus() === undefined
```

Comment:
Please also change `response.getMempoolStatus()` as explained in the comment above and update the `LibraMempoolTransactionStatus` enum to account for the -1 value.

Reply:
Ok.